airgun/views/os.py

```diff
@@ -9,6 +9,9 @@
 class OperatingSystemView(BaseLoggedInView):
     title = Text("//h1[text()='Operating systems']")
     new = Text("//a[contains(@href, '/operatingsystems/new')]")
+    delete = GenericLocatorWidget(
+        "//span[contains(@class, 'btn')]/a[@data-method='delete']")
+    navigate_locator = "//a[@id='menu_item_operatingsystems']"
```

Comment:
This locator is a leftover of some old commit, probably appeared during conflicts resolution, please remove it.